README.md

```diff
@@ -115,7 +128,40 @@ Add a different test function to the `src/test_functions/` directory,
 im- and export it in `src/test_functions/index.js` and specify its name in
 `testFunctionName` in `index.js`.
 
-## Running the unit tests
+## Local development
+
+To avoid the build cycle of the Docker images, you can also develop
+locally.
+
+### Installing the dependencies
+
+	npm install
+
+### Run a rabbitmq instance
+
+	docker run -d --name amqp.test -p 5672:5672 rabbitmq
+
+
+### Running the workers
+
+You have to set the environment variables `QUEUE_URL` (e.g. `amqp://localhost`), `TB_SECRET` and `TB_KEY` (authentication data for testingbot) before running the scripts.
+
+	npx ts-node metadata_worker.ts
+	npx ts-node screenshot_worker.ts
+
+Instead of the full-fledged screenshot worker you can also run
+`simple_worker.js` which echoes the data it receives.
+
+
+### Running the screenshot tool
+
+	npx ts-node queue_screenshots.ts -c path/to/campaign_info.toml <CAMPAIGN_NAME>
```

Comment:
It's not easier to run but something like this would make it more consistent if the other commands were in the paclage.json:

`npm run screenshot --info-path=path/to/campaign_info.toml --campaign=campaign`

Reply:
You'd have to stick another `--` in there to pass the parameters, leading to potential confusion (if you leave it out because of a copy-paste mistake or if you think it's a type in the docs):

`npm run screenshot -- --info-path=path/to/campaign_info.toml --campaign=campaign`

I think avoiding consistency is better in this case.